packages/botonic-plugin-flow-builder/src/action/payload.ts

```diff
@@ -19,3 +26,46 @@ export async function getContentsByPayload({
 
   return []
 }
+
+async function resolveRatingPayload(
+  context: FlowBuilderContext
+): Promise<FlowContent[]> {
+  const { cmsApi, flowBuilderPlugin, request } = context
+
+  if (!request.input.payload) {
+    return []
+  }
+
+  const id = request.input.payload
+  const buttonId = id?.split(SEPARATOR)[1]
```

Comment:
alternatively, you can also do: `const [, buttonId] = id?.split(SEPARATOR) ?? []`